src/locales/ko/starter-select-ui-handler.json

```
@@ -2,6 +2,7 @@
   "confirmStartTeam": "이 포켓몬들로 시작하시겠습니까?",
   "confirmExit": "나가시겠습니까?",
   "invalidParty": "스타팅 포켓몬 파티에 적합하지 않습니다!",
+  "emptyParty": "포켓몬을 파티에 추가하려면 스페이스 바 또는 Z 키를 눌러주세요.\n시작하려면 엔터 키 또는 Start 버튼을 눌러주세요.",
```

```suggestion
  "emptyParty": "포켓몬을 파티에 추가하려면 스페이스 바 또는 Z 키를 눌러주세요.\n시작하려면 엔터 키 또는 시작 버튼을 눌러주세요.",
```
modified ``start`` -> ``시작``